lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -586,4 +601,33 @@ defmodule RealtimeWeb.RealtimeChannel do
     Logger.error("Start channel error: " <> error_msg)
     {:error, %{reason: error_msg}}
   end
+
+  defp assign_policies(false, db_conn, sub_topic, access_token, claims, socket) do
+    channel =
+      case ChannelsCache.get_channel_by_name(sub_topic, db_conn) do
+        {:error, :not_found} -> raise "Channel #{sub_topic} hasn't been initialized"
+        {:ok, channel} -> channel
+      end
+
+    authorization_context =
+      Authorization.build_authorization_params(%{
+        channel: channel,
+        headers: socket.assigns.headers,
+        jwt: access_token,
+        claims: claims,
+        role: claims["role"]
+      })
+
+    {:ok, socket} = Authorization.get_authorizations(socket, db_conn, authorization_context)
+
+    case socket.assigns.policies do
```

Comment:
In the case of an authorization error I was thinking we could respond here with the full policy generated.

Would make debugging your policies way easier probably. Unsure of security implications, but I can't think of any.

Reply:
that's why I was logging them out so the owner can know but the user isn't fully aware but honestly security by obfuscation isn't the best as it will never fully be useful 😅

Reply:
now I noticed I do not log out 🤦

Reply:
I returned an error with more information. I can send the policies afterwards if we want to